codecov.yml

```diff
@@ -1,7 +1,7 @@
 # config for codecov
 
 codecov:
-  token: eee2d1a8-f24b-401b-a211-19c10e987f53
+  token: df828ddb-54c2-4c3b-9959-0571877a1ea2
```

Comment:
@niyiyu is this needed here? I see that in the action you are fetching the token from a secret: `secrets.CODECOV_TOKEN`, which seems like the better way.

Reply:
I had the same question when I was testing the token. I will test removing this and see if it pull the same token from the repo secret/variable, maybe in another PR.